### ISSUE TITLE
A11y Documentation: Time Picker - Internationalization

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1400]
CENG_1968

https://react-magma.cengage.com/version/4.5.0/api/time-picker/#internationalization

### Issue:
The Internationalization section shows the time picker that uses a language other than English for AM and PM. If the purpose of this setting is to allow non-English speakers to set the time then the content should have the proper language attribute set. In addition, aria-labels for various inputs are in English.

### Suggestion:
Use the lang attribute to set the proper language, and ensure that all labels are in the appropriate language.

**Comment by silvalaura** [comment]
This is a problem with the Internationalization example. The code already supports using the i18n variable amButtonAriaLabel for the aria label.

**Comment by chris-cedrone-cengage** [comment]
![Width issue](https://github.com/user-attachments/assets/e6d8f1d1-938f-4cee-9196-b0f731ef52be)

Do we care if the Kanji is condensed due to the input width?

**Comment by silvalaura** [comment]
I think this is okay for now and addresses the original ticket. I opened a new ticket to address the width of the AM/PM field: https://github.com/cengage/react-magma/issues/1488
